subsys/net/l2/openthread/openthread.c

```diff
@@ -428,26 +438,30 @@ static int openthread_init(struct net_if *iface)
 		platformShellInit(ot_context->instance);
 	}
 
-	if (IS_ENABLED(CONFIG_OPENTHREAD_NCP)) {
+	if (IS_ENABLED(CONFIG_OPENTHREAD_COPROCESSOR)) {
 		otNcpInit(ot_context->instance);
 	}
 
-	otIp6SetEnabled(ot_context->instance, true);
+	if (!IS_ENABLED(CONFIG_OPENTHREAD_COPROCESSOR_RCP)) {
+		if (!IS_ENABLED(CONFIG_OPENTHREAD_RAW)) {
```

Comment:
The `if (!IS_ENABLED(CONFIG_OPENTHREAD_RAW)) {` condition was already removed on master (it was a bug), please do not re-introduce it.

Reply:
Done.